packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -21,6 +21,89 @@ const validManageableStates = new Set([
   undefined // not part of a package
 ]);
 
+const STANDARDVALUESET_FULLNAME = 'StandardValueSet';
```

Comment:
Can we reuse `standardValueSet.fullnames` from `source-deploy-retrieve` (`import { standardValueSet } from '@salesforce/source-deploy-retrieve/lib/src/registry'`) here instead

Reply:
addressed.

thanks for checking it out this quick :)